tracer/test/Datadog.Trace.Tests/ActivityTests.cs

```diff
@@ -0,0 +1,229 @@
+// <copyright file="ActivityTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if !NETFRAMEWORK
```

Comment:
We do support .NET Framework, because we instrument the library no matter the runtime version. I've added the test support in: https://github.com/DataDog/dd-trace-dotnet/pull/2446/commits/763251ad242e184ec59dfad10f57f99598a4c831